UKHomeOffice/nmsw-ui
# NMSW-1372 Link to ETA page from Dashboard

# Ticket



----

## AC

Add more info section with link to eta page

----

## To test

- sign in
- you should see a more information section with a link to the ETA page


----

## Notes
